congress/metadata/G000581.yaml

```diff
@@ -0,0 +1,2 @@
+name: Facebook
+link: https://www.facebook.com/votevicente/
```

Comment:
I'm not sure why this is two separate bioguide IDs -- the images for G000580 and G000581 are both for Vicente Gonzalez. Also, this one is from a campaign Facebook page.

https://www.facebook.com/votevicente/